anaconda.py

```diff
@@ -603,10 +603,10 @@ def _earlyExceptionHandler(ty, value, traceback):
     from pykickstart import constants as pykickstart_constants
```

Comment:
Also "*Only TEXT and GUI mode is now used*" "*Only TEXT and GUI modes are now used*"